src/test/java/org/mitre/synthea/helpers/FactTableTest.java

```diff
@@ -37,4 +37,14 @@ public void testFactTable() {
     Assert.assertTrue(output.contains(he + ",He"));
   }
 
+  @Test
+  public void testGetFactByKey() {
+    // Setup a basic fact table with a header.
+    FactTable table = new FactTable();
+    table.setHeader("ID,KEY,NAME,DESCRIPTION");
+    // Insert some facts.
+    int h = table.addFact("H", "Hydrogen,Highly flammable gas");
+    Assert.assertNull(table.getFactByKey("Iwcz&T&5+;N4Sb)kE+#P"));
+  }
```

Comment:
```suggestion
    Assert.assertEquals("Hydrogen,Highly flammable gas", table.getFactByKey("H"));
  }
```

This will add a positive test as well as the negative test. You could break them out into separate unit tests if you wanted.